apps/jobs/urls.py

```diff
@@ -9,6 +9,10 @@
     url(r'challenge/(?P<challenge_id>[0-9]+)/'
         r'challenge_phase/(?P<challenge_phase_id>[0-9]+)/submission/',
         views.challenge_submission, name='challenge_submission'),
+    url(r'challenge/(?P<challenge_id>[0-9]+)/'
+        r'challenge_phase/(?P<challenge_phase_id>[0-9]+)/participant_team/(?P<participant_team_id>[0-9]+)/'
+        r'get_remaining_submission/',
+        views.remaining_submission, name='remaining_submission'),
```

Comment:
@deshraj : May I know why are we using hyphen instead of underscore in urls. As far as all the urls are concerned in the project, I think we are using underscore.